### ISSUE TITLE
Receive deprecated warnings at GH workflow step for version trilom/file-changes-action@v1.2.4

### ISSUE THREAD

**Issue by conapki** [trilom/file-changes-action#141]
**Describe the bug**
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

In our workflow we did fixed the 'set-output' command with help of https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

But still wherever we have used trilom/file-changes-action@v1.2.4 in our workflow steps we receive the above warnings.

**Workflow**
If applicable, provide a workflow file to help explain your problem.

**Expected behavior**
Is it something that new version of this repo will fix the warnings list that we get in GH action run ? Please help on same.

**Additional context**
![image](https://user-images.githubusercontent.com/99417523/201876605-e127e813-6310-444e-91b5-09023398fe7d.png)

**Comment by conapki** [comment]
Hi, Any updates to fix the set-output deprecated warnings?

**Comment by lmmentel** [comment]
It seems this repo got abandoned...

**Comment by honeyverma** [comment]
Anybody knows if they are going to update it or do we need to use another plugin? if yes, can anybody recommend the new one?

**Comment by nsandary** [comment]
Yep, looks like we'll need a new one. Maybe someone responsible should fork this repo and maintain it.

**Comment by 808sAndBR** [comment]
I have added a fix for this and opened a PR here: https://github.com/trilom/file-changes-action/pull/148

Given that I opened a documentation PR a while back and it didn't get merged, I have also gone ahead and created a fork with a release of this fix: 
https://github.com/808sAndBR/file-changes-action/releases/tag/v1.2.5

I don't plan to actively maintain or be especially "responsible" but it is working for my stack and anyone that wants to update to `uses: 808sAndBR/file-changes-action@v1.2.5` is welcome to. If @trilom merges this change into main I will be pointing my actions back to their's but will leave my fork and it's release as is.

**Comment by 808sAndBR** [comment]
Also I am really more of a pythonista that has dipped their toes into a few hours of JS/TS so if anyone has feedback on anything I should have done differently with this change, I am happy to update and stamp a new release.

**Comment by john-jerome** [comment]
this workflow is actively developed and doing pretty much the same thing:
https://github.com/tj-actions/changed-files

**Comment by santicarmo31** [comment]
Any update on this?

**Comment by jpmckinney** [comment]
I switched to https://github.com/tj-actions/changed-files - it works (even better).
